src/api/functions.jl

```diff
@@ -5402,4 +5403,3 @@ function h5fd_stdio_init()
     var"#status#" < 0 && @h5error("Error initializing file driver")
     return var"#status#"
 end
-
```

Comment:
This is slightly annoying. I'm not sure what the appropriate fix is.

Reply:
Probably an editor mistake on my side (not the formatter).

Reply:
Fixed.